docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -5,41 +5,47 @@
 Adding modules and plugins locally
 **********************************
 
-.. contents::
-   :local:
+The easiest, quickest, and the most popular way to extend Ansible is to use a module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by embedding it in a role and publishing it on Ansible Galaxy. If you are using roles on Ansible Galaxy, then you are already using local modules and plugins without realizing it.
```

Comment:
The first sentence here, as changed, is not quite accurate. Modules and plugins are the building blocks of Ansible behavior. This page shows users how to get a **local** plugin or module to work.
```suggestion
The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by embedding it in a role and publishing it on Ansible Galaxy. If you are using roles on Ansible Galaxy, then you are already using local modules and plugins without realizing it.
```